h/static/scripts/widget-controller.js

```diff
@@ -327,7 +327,14 @@ module.exports = function WidgetController(
     loadAnnotations(crossframe.frames);
 
     // When switching groups, reset back to the annotations tab
-    annotationUI.selectTab(uiConstants.TAB_ANNOTATIONS);
+    // if the selected tab is an orphan. Otherwise carry the
+    // tab selection over to the changed group.
+    var selectedTab = annotationUI.getState().selectedTab;
+    if (selectedTab === uiConstants.TAB_ORPHANS) {
+      annotationUI.selectTab(uiConstants.TAB_ANNOTATIONS);
+    } else {
+      annotationUI.selectTab(selectedTab);
```

Comment:
This line is a no-op, or at least there is a problem somewhere else if it isn't, so it should be possible to remove the `else` block here.